packages/php-datatypes/src/Definition/Bigquery.php

```diff
@@ -76,7 +76,17 @@ class Bigquery extends Common
     public const TYPE_STRING = 'STRING'; // STRING(L) L is a positive INT64 value
 
     public const TYPE_STRUCT = 'STRUCT';
-
+    // Map of types available in SQL api but not supported by REST API
+    // Most of them are aliases for other types
+    public const REST_API_TYPES_MAP = [
+        self::TYPE_INT => self::TYPE_INT64,
```

Comment:
Ok, ale co `ARRAY` a `INTERVAL` ?

<img width="514" alt="image" src="https://github.com/keboola/storage-backend/assets/13363655/1c1f959d-d2db-4548-bc7f-a6f13de85511">

myslim, ze oba tyto typy nemame dobre uchopene v connection. array by mel prijit s definici a interval taky. Imho bychom to meli zakazat na request objectu. Toto bych ale shipnul a na ten array a interval zalozil dalsi task

Reply:
array je repeated record a interval nemá dokumentaci pro REST API ani, takže sem ho zatím přeskočil

Reply:
jo, ale kdyz to sem dorazi, tak to nekde nejak neplanovane vyhuci. Takze bychom to meli imho odchytit uz nekde v connection. Takze to bude chtit nejakou dalsi praci

Reply:
https://keboola.atlassian.net/browse/BIG-238